pkg/controller/core/clusterqueue_controller.go

```diff
@@ -107,16 +110,17 @@ func (r *ClusterQueueReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		}
 	}
 
-	status, err := r.Status(&cqObj)
-	if err != nil {
-		log.Error(err, "Failed getting status from cache")
-		return ctrl.Result{}, err
-	}
-
-	if !equality.Semantic.DeepEqual(status, cqObj.Status) {
-		cqObj.Status = status
-		err := r.client.Status().Update(ctx, &cqObj)
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+	newCQObj := cqObj.DeepCopy()
+	if r.cache.ClusterQueueActive(newCQObj.Name) {
+		msg := fmt.Sprintf("ClusterQueue %q can admit new workloads", newCQObj.Name)
```

Comment:
No need to complicate the message if this is a condition of the ClusterQueue itself.

```suggestion
		msg := "Can admit new workloads"
```

Reply:
Done.